### ISSUE TITLE
Routing error

### ISSUE THREAD

**Issue by rachary00** [magical-eda/MAGICAL#54]
I get the following error:
<img width="1269" alt="Screen Shot 2022-05-12 at 5 11 38 PM" src="https://user-images.githubusercontent.com/9872021/168176367-f6c67529-1a96-484e-bdcd-4fb1320549e9.png">

I am using a simple custom circuit and the PDK files are all the mock files provided in the GitHub. I am not sure what this error is exactly. Please help!

**Comment by krzhu** [comment]
Looked at the log. It might because the router cannot find a access point for a pin. We first find the overlapping points of a pin shape and the routing grid. 

@baloneymath Do you know anything about it?

**Comment by baloneymath** [comment]
It should be caused by some pins that were not on-grid and thus no valid access point exists. @rachary00 could you check if the pins in your circuit are correct?

**Comment by rachary00** [comment]
yes the pins in schematic file look fine, is there any place else that you want me to check them?

**Comment by baloneymath** [comment]
@rachary00 could you check if the layout after placement (before running routing) is correct? This problem seems that the layers are not initialized correctly.

**Comment by rachary00** [comment]
It looks correct, not sure what to check exactly, but all the transistor cells are in the right location
